src/builder/create_embed.rs

```diff
@@ -198,6 +198,13 @@ impl CreateEmbed {
         super::check_overflow(length, crate::constants::EMBED_MAX_LENGTH)
             .map_err(|overflow| Error::Model(ModelError::EmbedTooLarge(overflow)))
     }
+
+    /// Retrieves the inner [`Embed`].
+    #[inline]
+    #[must_use]
+    pub fn get(self) -> Embed {
```

Comment:
Methods that consume and convert a type to another should follow the `into_*` convention:
```suggestion
    pub fn into_inner(self) -> Embed {
```